instruments/toptica/topmode.py

```diff
@@ -137,11 +139,23 @@ def enable(self):
         @enable.setter
         def enable(self, newval):
             if not isinstance(newval, bool):
-                raise TypeError(
-                    "Laser emmission must be a boolean, got: {}".format(newval))
+                raise TypeError("Emission status must be a boolean, got: {}".
```

Comment:
Let's not line break on the `.`, instead do it just before the word "got" for code readability